hooks/usePlayerActions/normalizeGameStreetActions.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  GameStreetAction,
+  GameStreetActionType,
+} from "@@/models/GameSituation";
+import getCyclicNextPlayerIndexOf from "./getCyclicNextPlayerIndexOf";
+
+export default function normalizeGameStreetActions({
+  actions,
+  activePlayerIndexes,
+  isPreflop,
+}: {
+  actions: GameStreetAction[];
+  activePlayerIndexes: Set<number>;
+  isPreflop: boolean;
+}): GameStreetAction[] {
+  const nextActions = [...actions];
+  let index = 0;
+  let playerIndexToAct = isPreflop ? 2 : Math.min(...activePlayerIndexes);
+  let nonBetStreak = 0;
+
+  while (nonBetStreak < activePlayerIndexes.size - 1) {
+    // Add action when the action which is supposed to be there doesn't exist.
+    if (
+      actions[index] === undefined ||
```

Comment:
It would be better if it's checked whether the index is in range or out, instead of trying to access elements